bin/medic.js

```diff
@@ -19,18 +19,44 @@
  * under the License.
  */
 
-"use strict";
+/* jshint node: true */
+
+// "use strict";
 
 // node dependencies
-var fs = require("fs");
-var os = require("os");
+var fs   = require("fs");
+var os   = require("os");
+var path = require("path");
 
 // external dependencies
 var shelljs  = require("shelljs");
 var optimist = require("optimist");
 
+// internal dependencies
+var testwait  = require("./lib/testwait");
+var testcheck = require("./lib/testcheck");
+
 // constants
-var MAX_REMOVAL_ATTEMPTS = 3;
+var MAX_REMOVAL_ATTEMPTS    = 3;
+var CORDOVA_MEDIC_DIR       = "cordova-medic";
+var DEFAULT_APP_PATH        = "mobilespec";
+var DEFAULT_APP_ENTRY       = "index.html";
+var ANDROID_TIMEOUT         = 120000;
```

Comment:
Android timeout is really, really long if minute is 60...  Maybe rewrite ANDROID_TIMEOUT in context of ONE_MINUTE?